service/src/main/java/bio/terra/workspace/service/resource/ValidationUtils.java

```diff
@@ -65,6 +65,9 @@
   public static final Pattern RESOURCE_NAME_VALIDATION_PATTERN =
       Pattern.compile("^[a-zA-Z0-9][-_a-zA-Z0-9]{0,1023}$");
 
+  public static final Pattern GIT_URL_PATTERN =
```

Comment:
Is this a URI more than a URL?

Reply:
does check for git specific elements, https (https://<host>/<organization>/<repo>.git) or ssh url (git@github.com:<organization>/<repo>.git)